### PR TITLE
Br staging

### PR DESCRIPTION
**Pull request by Humairaa127**
Staging customization for Safernet. 
- Does not include platform localisation, only fields.
- @murilovmachado @nickhurlburt please see for the payload definition

### CODE REVIEW

File: plugin-hrm-form/src/formDefinitions/br-v1/caseForms/NoteForm.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "name": "note",
+    "label": "Notas",
+    "type": "textarea",
+    "placeholder": "Type here to add note...",
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/caseForms/NoteForm.json:6]: not translated

File: plugin-hrm-form/src/formDefinitions/br-v1/caseForms/PerpetratorForm.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      { "value": "25", "label": "25" },
+      { "value": ">25", "label": ">25" },
+      { "value": "Desconhecido", "label": "Desconhecido" }
+    ],
+    "required": { "value": true, "message": "RequiredFieldError" }
+  },
+  {
+    "name": "relationshipToChild",
+    "type": "select",
+    "label": "Relationship to child",
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/caseForms/PerpetratorForm.json:55]: not translated

File: plugin-hrm-form/src/formDefinitions/br-v1/caseForms/PerpetratorForm.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    "options": [
+      { "value": "", "label": "" },
+      { "value": "Cuidador(a)", "label": "Cuidador(a)" },
+      { "value": "Par/amigo(a)", "label": "Par/amigo(a)" },
+      { "value": "Grandparent", "label": "Grandparent" },
+      { "value": "Neighbour", "label": "Neighbour" },
+      { "value": "Parent", "label": "Parent" },
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/caseForms/PerpetratorForm.json:62]: not translated

File: plugin-hrm-form/src/formDefinitions/br-v1/caseForms/ReferralForm.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "name": "date",
+    "type": "date-input",
+    "label": "Data",
+    "required": { "value": true, "message": "RequiredFieldError" }
+  },
+  {
+    "name": "referredTo",
+    "label": "Referred To",
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/caseForms/ReferralForm.json:10]: not translated

File: plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CallerInformationTab.json [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+      { "value": "Desconhecido", "label": "Desconhecido" }
+    ],
+    "required": { "value": true, "message": "RequiredFieldError" }
+  },
+  {
+      "name": "relationshipToChild",
+      "label": "Relacionamento com a criança",
+      "type": "select",
+      "options": [
+        { "value": "Unknown", "label": "" },
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CallerInformationTab.json:67]: not translated

File: plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CallerInformationTab.json [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+        { "value": "Cuidador(a)", "label": "Cuidador(a)" },
+        { "value": "Friend", "label": "Friend" },
+        { "value": "Grandparent", "label": "Grandparent" },
+        { "value": "Neighbour", "label": "Neighbour" },
+        { "value": "Parent", "label": "Parent" },
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CallerInformationTab.json:72]: not translated

File: plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CallerInformationTab.json [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+    "required": { "value": true, "message": "RequiredFieldError" }
+  },    
+  {
+    "name": "state",
+    "label": "Estado/País",
+    "type": "input"
+  },
+  {
+    "name": "city",
+    "label": "CIdade",
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CallerInformationTab.json:90]: Typo in **CI**dade, it should be **Ci**dade. (lowercase "i").

File: plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CaseInformationTab.json [hunk 8]
```diff
@@ -0,0 +1,58 @@
+[
+    {
+      "name": "callSummary",
+      "label": "Contact Summary",
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CaseInformationTab.json:4]: not translated

File: plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CaseInformationTab.json [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+      "type": "textarea",
+      "required": { "value": true, "message": "RequiredFieldError" }
+    },
+    {
+      "name": "actionTaken",
+      "label": "Providências tomadas",
+      "type": "select",
+      "options": [
+        { "value": "Unknown", "label": "" },
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/CaseInformationTab.json:13]: not translated

File: plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/ChildInformationTab.json [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+    "required": { "value": true, "message": "RequiredFieldError" }
+  },
+  {
+    "name": "state",
+    "label": "Estado/País",
+    "type": "input"
+  },
+  {
+    "name": "city",
+    "label": "CIdade",
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/br-v1/tabbedForms/ChildInformationTab.json:85]: Same type in **CI**dade. It should be lower case "i".

**Comment by murilovmachado** [comment]
I won't merge this just yet, there's some code that needs to be fixed before these translations work fine with our app. Working on them.

**Comment by Humairaa127** [comment]
Thanks Murilo! Yes these were translations we didn't ask Juliana for so
just left them as English to first see if they want the values at all.

**Pull request merged**